src/Components/Controls/Controls.js

```diff
@@ -50,22 +51,23 @@ export function Controls() {
         <Col md={6}>
           <Form className="mb-3">
             <fieldset>
-              <legend className="control-title">
+              <legend className={styles['control-title']}>
```

Comment:
It's better to follow this syntax
```suggestion
              <legend className={styles.controlTitle}>
```

Reply:
It's definitely better, the kebab case made changing to modules really annoying. I originally wrote it that way because I'm using Bootstrap and I didn't really want to use different conventions. Changing it seemed like it would not be worth it, but I might change it anyway (especially if I stop using Bootstrap).